### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
[DEMO LINK](https://Anton-Zhytonbaiev.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/components/PostInfo/PostInfo.tsx [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+        )}
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length < 1
+        ? <b data-cy="NoCommentsMessage">No comments yet</b>
+        : <CommentList comments={comments} />}
```
Comment (NVyrsta) [inline src/components/PostInfo/PostInfo.tsx:38]:
```suggestion
      {comments.length
        ? <CommentList comments={comments} />
        : <b data-cy="NoCommentsMessage">No comments yet</b>}
```

File: src/components/PostList/PostList.tsx [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (<PostInfo key={post.id} post={post} />))}
```
Comment (NVyrsta) [inline src/components/PostList/PostList.tsx:11]:
```suggestion
    {posts.map(post => (
      <PostInfo key={post.id} post={post} />))}
```

**Review by NVyrsta — APPROVED** [review]
Good job ;)

File: src/App.tsx [hunk 3]
```diff
@@ -2,108 +2,28 @@ import React from 'react';
...
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
-
-          <p>
-            {' Posted by  '}
+const getUser = (userId: number) => {
+  return usersFromServer.find(user => userId === user.id) || null;
+};
```
Comment (alexandra-protyanova) [inline src/App.tsx:13]:
```suggestion
const getUser = (userId: number) => usersFromServer.find(user => userId === user.id) || null;
```

**Review by alexandra-protyanova — APPROVED** [review]
Well done 👍

**Pull request closed without merging**